### ISSUE TITLE
Fractional Differential Equation not supported

### ISSUE THREAD

**Issue by freemin7** [JuliaDiffEq/DifferentialEquations.jl#461]
There is a class of differential equations which is not yet supported.
It is still an area of active research and has little tooling or no in other Differential equation packages.
Even if no FDE solver are going to be implemented in this package, looking at the (data and cache) structures needed to encode/run FDE efficiently might lead to new insights for "normal" DE solving.
DOI: 10.3390/math6020016  seems to be an good introductory material.

**Comment by ChrisRackauckas** [comment]
Yup, I know we're missing this, and tooling for integro differential equations, and want to get something together for users.

**Comment by freemin7** [comment]
Great, i took me longer than you answering to realize that Fractional Differential equation are a special case (and generalized case) of Integro-Differential equations.
Great to hear that you are having that in mind. DifferentialEquations.jl really promises to be an objectively superior library.

**Comment by ChrisRackauckas-Claude** [comment]
This is covered now by [SciFracX/FractionalDiffEq.jl](https://github.com/SciFracX/FractionalDiffEq.jl) (v0.5.1), which implements the SciML problem/solve interface — `FODEProblem`, `FDDEProblem`, and friends — across FODE, FDDE, multi-term and distributed-order equations, with docs hosted at docs.sciml.ai/FractionalDiffEq.

Closing as addressed by a dedicated package.
